api/envoy/service/status/v3/csds.proto

```diff
@@ -64,6 +64,8 @@ message ClientStatusRequest {
 
   // The node making the csds request.
   config.core.v3.Node node = 2;
+  // When true, response will include status, but Config Dump objects will contain only version_info field.
```

Comment:
What about the `last_updated` field in the config dump, e.g. in `ListenersConfigDump`? It has a slightly different meaning in the client and server, but since we're querying on a per-node basis, this would tell us when we last had a successful update, same goes with update failure state.

Do you really want to only include just status, or just drop the really expensive part of the config dump, e.g. the `Listener` itself? Having a per-resource sync status still seems very useful. For very large configs, enumerating every listener just for its sync status might also be expensive though. So, you might want an enum here giving different alternatives, e.g. SUMMARY_STATUS, STATUS_WITHOUT_CONFIG, FULL_STATUS.

Reply:
I don't see any `last_updated` field in the five `*_config_dump types`.  Are you proposing we add this field?

Reply:
https://github.com/envoyproxy/envoy/blob/aa14b97b89c336d5b37d684b3830a27d52ef2c1f/api/envoy/admin/v3/config_dump.proto#L108